src/sentry/integrations/slack/message_builder/incidents.py

```diff
@@ -30,7 +30,7 @@ def __init__(
         notification_uuid: str | None = None,
     ) -> None:
         """
-        Builds an incident attachment for slack unfurling.
+        Builds an incident attachment when a metric alert fires or is resolved.
```

Comment:
I confirmed with Cathy that this builder is NOT for unfurling, that's the `SlackMetricAlertMessageBuilder`